man/man1/registercloudguest.1

```diff
@@ -3,18 +3,18 @@
 .\"
 .TH registercloudguest.1
 .SH NAME
-registercloudguest \- Register a SUSE Linux Enterprise on-demand instance
+registercloudguest \- Register a SUSE Linux Enterprise instance
 .SH SYNOPSIS
 .B registercloudguest [option]
 .SH DESCRIPTION
 .B registercloudguest
-Register a SUSE Linux Enterprise on-demand instance with the closest, in most
-cases region local SUSE operated update infrastructure. The client will reach
-out to a so called region server to obtain information about the region local
+Register a SUSE Linux Enterprise instance with the closest, in most
+cases region local, SUSE operated update infrastructure. The client will reach
+out to a so called region server to obtain information about the closest
```

Comment:
'closest (by default)' ?

Same for line 11
One can run `registercloudguest --smt-ip <IP_OF_NOT_THE_CLOSEST_UPDATE_SERVER>...`

Or maybe we don't care/mind too much that use case